pkgs/development/python-modules/ansible-lint/default.nix

```
@@ -13,27 +13,25 @@
 , wcmatch
 , enrich
 , python
+, tenacity
+, flaky
+, yamllint
 }:
 
 buildPythonPackage rec {
   pname = "ansible-lint";
-  version = "5.0.2";
+  version = "5.0.8";
   disabled = isPy27;
   format = "pyproject";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-vgt/KqNozTPaON/I19SybBZuo7bbl3Duq5dTBTMlj44=";
+    sha256 = "sha256-tnuWKEB66bwVuwu3H3mHG99ZP+/msGhMDMRL5fyQgD8=";
   };
 
-  postPatch = ''
-    substituteInPlace src/ansiblelint/file_utils.py \
-      --replace 'raise RuntimeError("Unable to determine file type for %s" % pathex)' 'return "playbook"'
-  '';
-
   buildInputs = [ python ];
 
-  propagatedBuildInputs = [ ansible enrich pyyaml rich ruamel-yaml wcmatch ];
+  propagatedBuildInputs = [ ansible enrich pyyaml rich ruamel-yaml wcmatch tenacity flaky yamllint ];
 
   checkInputs = [ pytestCheckHook pytest-xdist git ];
```

```suggestion
  checkInputs = [
    pytestCheckHook
    pytest-xdist
    git
  ];
  
  pytestFlagsArray = [
    "--numprocesses" "auto"
  ];
```

Without this argument xdist doesn't parallelize the test suite.